.github/workflows/releaser-master.yml

```diff
@@ -21,8 +24,25 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+
+      - name: Determine release mode
+        id: release-mode
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "mode=test" >> $GITHUB_OUTPUT
+            echo "args=build --snapshot --skip=before" >> $GITHUB_OUTPUT
```

Comment:
I'm not familiar with goreleaser but, but I see on their [page dedicated to dry-run](https://goreleaser.com/quick-start/#dry-run) that among various options, they mention this one: ⁠
```bash
goreleaser release --skip=publish
```

Instinctively, I think this would involve all the `release` steps except for the publishing, which might help catch more errors in the PR before they happen on master. WDYT?

Reply:
@aeddi it should work with `build --snapshot --skip=before` if you want to achieve exclusively the binaries.
 
Given that they are not published, I think the goal is testing if building of binaries / docker images actually works